packages/action/package.json

```diff
@@ -0,0 +1,24 @@
+{
+	"name": "@release-drafter/action",
+	"version": "6.0.0-beta.1",
+	"description": "",
+	"main": "dist/index.js",
+	"type": "module",
+	"scripts": {
+		"build": "ncc build src/index.ts --target es2022 --source-map --license licenses.txt",
```

Comment:
Is building with vercel's ncc part of how GithubActions should be built ?

I feel like having a single build system in the monorepo would be the way to go (tsup, esbuild, rollup or other)

Also, since this package is the only one using ncc, I feel like the corresponding `@vercel/ncc` dev-dependency should be here instead of the root package.json file